app/views/setup/bootstrap.html.slim

```diff
@@ -0,0 +1,18 @@
+h1 Confirm bootstrap
+
+= form_for :settings, url: setup_bootstrap_path, method: :post do |f|
+  .panel.panel-default
+    .panel-heading
+      h3.panel-title Cluster specific settings
+    .panel-body
+      .form-group
+        = f.label :apiserver, "External Kubernetes API server FQDN"
+        .input-group
+          = f.text_field :apiserver, value: (Pillar.value(pillar: :apiserver) || Minion.find_by(role: Minion.roles[:master]).fqdn), class: "form-control", required: true
```

Comment:
Move `(Pillar.value(pillar: :apiserver) || Minion.find_by(role: Minion.roles[:master]).fqdn)` to @apiserver in controller?

Reply:
I think this would make the view more clear, which is good. We should do this on the first page of the setup too for consistency. Will update PR.

Reply:
PR has been updated with this @vitoravelino